core/src/scss/elements/nav/_nav.scss

```diff
@@ -0,0 +1,15 @@
+@charset "UTF-8";
+//
+// Navigation 
+//
+// Default styling for navigation or menu elements.
+//
+
+
+nav {
+  a {
+    &:focus {
+      box-shadow: none;
```

Comment:
So for navigation or menu elements, we only want an underline?

Reply:
Good question @jenbreese. I don't want to have a box shadow on `&:focus`. 
I want it at `&:focus-visible`.